src/Integration.Vsix/Delegates/SonarAnalyzerWorkflowBase.cs

```diff
@@ -58,10 +59,24 @@ protected SonarAnalyzerWorkflowBase(Workspace workspace)
         }
 
         internal /* for testing purposes */ bool ShouldRegisterContextActionWithFallback(
-            IEnumerable<DiagnosticDescriptor> descriptors) =>
-            ShouldRegisterContextAction(descriptors)
-            // Fallback using SonarWay
-            ?? descriptors.Any(d => d.CustomTags.Contains(DiagnosticTagsHelper.SonarWayTag));
+            IEnumerable<DiagnosticDescriptor> descriptors)
+        {
+            // Let's disable all the utility analyzers under SonarLint
+            // - Note 1:
+            // There is already some logic within the analyzer to disable the execution when the output-cs/vbnet folder is not
+            // found which I decided to skip here.
+            // - Note 2:
+            // The CPD could be useful but is also disabled at the moment
+            if (descriptors.Any(d => d.Id.StartsWith("S9999-")))
```

Comment:
On balance, it might be better to go with your alternative suggestion of checking for the "NotConfigurable" tag. It's a more general solution, and if a rule is marked as "NotConfigurable" then it seems reasonable that this code should respect that setting.